ArduPlane/Log.cpp

```diff
@@ -470,8 +471,8 @@ const struct LogStructure Plane::log_structure[] = {
       "CMDA", "QHBBBBffffiifB",    "TimeUS,CId,TSys,TCmp,cur,cont,Prm1,Prm2,Prm3,Prm4,Lat,Lng,Alt,F", "s---------DUm-", "F---------GGB-" }, 
     { LOG_CMDH_MSG, sizeof(log_CMDI),     
       "CMDH", "QHBBBBffffiifB",    "TimeUS,CId,TSys,TCmp,cur,cont,Prm1,Prm2,Prm3,Prm4,Lat,Lng,Alt,F", "s---------DUm-", "F---------GGB-" }, 
-
 };
+#endif
```

Comment:
this isn't good it breaks the #else at L535

Reply:
#else
const struct LogStructure Plane::log_structure[]{};
#endif

Reply:
Fixed - moved to using `HAL_LOGGING_ENABLED` everywhere rather than a mix of `HAL_LOGGING_ENABLED` and `LOGGING_ENABLED`